app/presenters/comment_presenter.rb

```diff
@@ -19,11 +20,26 @@ def as_api_response
       author:           PersonPresenter.new(author).as_api_json,
       created_at:       created_at,
       mentioned_people: build_mentioned_people_json,
-      reported:         current_user.present? && reports.where(user: current_user).exists?
+      reported:         current_user.present? && reports.where(user: current_user).exists?,
+      interactions:     build_interactions_json
+    }
+  end
+
+  def  build_interactions_json
+    {
+      likes:            as_api(likes),
```

Comment:
Layout/HashAlignment: Align the keys and values of a hash literal if they span more than one line.